### ISSUE TITLE
Uninstall button

### ISSUE THREAD

**Issue by Romanitho** [Romanitho/Winget-Install-GUI#21]
### Discussed in https://github.com/Romanitho/Winget-Install-GUI/discussions/20

<div type='discussions-op-text'>

<sup>Originally posted by **KnifMelti** May  6, 2022</sup>
Why not have that too in the main form? :=)</div>

**Comment by KnifMelti** [comment]
Great!
But, one can mix the installed with not installed by first **List installed** and then **Search for an app** and **Add to List** making the **Install** button go through all...

**Comment by Romanitho** [comment]
Exact. That's why I wanted a separate tab initially. But in fact, it can be interesting to use the same list to export the installed apps of a computer, and add more in that list if you want.  
If you missclick on Install, it will go through all apps, but will not try to reinstall already installed ones.

What did you thing about to get it better ? :)

**Comment by KnifMelti** [comment]
Ah, I see, for the export of the list. 
Nice!
Maybe disable the install button when the list gets populated with installed apps and enable it when cleared again?

**Comment by KnifMelti** [comment]
...and a Clear button to quickly empty rhe list...

**Comment by Romanitho** [comment]
Not easy to manage. If a user add apps to that list, should we unclear the install button ? Normally, you should not use WiGui that much (for majority of us). We should not install/uninstall apps that often (expect for our job :p).

**Comment by Romanitho** [comment]
But adding a "Clear all" button might be useful, as I didn't found a way to Ctrl+A the App List (would be nice to select all and click "Remove" Button, but I don't know how to do).
So a "Clear" button might be the solution :)

**Comment by KnifMelti** [comment]
Ctrl+A is a better way, less button clutter.

**Comment by KnifMelti** [comment]
...then act on the Del key too to erase 😉

**Comment by Romanitho** [comment]
probably change the "list installed" from `Winget list` to `Winget export` to get a json export of winge apps only.
